pkg/volume/flexvolume/mounter-defaults.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flexvolume
+
+import (
+	"k8s.io/kubernetes/pkg/volume"
+)
+
+type mounterDefaults flexVolumeMounter
+
+// SetUpAt is part of the volume.Mounter interface.
+// This implementation relies on the attacher's device mount path and does a bind mount it's not dir.
+func (f *mounterDefaults) SetUpAt(src, dir string, fsGroup *int64) error {
+	err := doMount(f.mounter, src, dir, "auto", []string{"bind"})
+	if err != nil {
+		return err
+	}
+	if !f.readOnly {
+		volume.SetVolumeOwnership((*flexVolumeMounter)(f), fsGroup)
+	}
+	return nil
+}
+
+// Returns the default volume attributes.
+func (f *mounterDefaults) GetAttributes() volume.Attributes {
+	return volume.Attributes{
+		ReadOnly:        f.readOnly,
+		Managed:         !f.readOnly,
+		SupportsSELinux: true,
```

Comment:
Flexvolume wants this support too :) I worked around SELinux issues in our internal plugin by going side band and pulling the security context. Ideally, we want to pass the complete securitycontex to flexvolume as an option, but I am saving it for now from this PR.